<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CircleTipsWidget</class>
 <widget class="QWidget" name="CircleTipsWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1920</width>
    <height>1080</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>80</y>
     <width>1500</width>
     <height>920</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">#groupBox{background: #FFFFFF;
border-radius: 8px;}</string>
   </property>
   <property name="title">
    <string/>
   </property>
   <widget class="QPushButton" name="confirm_Btn">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>850</y>
      <width>1501</width>
      <height>71</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>黑体</family>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{background: #0D9DDB;border-radius: 8px;color:white;}
QPushButton:hover{background: #1EAACF;border-radius: 8px;color:#333333;}</string>
    </property>
    <property name="text">
     <string>确认</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="step1_GroupBox">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>130</y>
      <width>400</width>
      <height>400</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">#step1_GroupBox{background: #E4F4FC;
border-radius: 12px;}</string>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QLabel" name="step1Picture_Label">
     <property name="geometry">
      <rect>
       <x>14</x>
       <y>0</y>
       <width>376</width>
       <height>400</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </widget>
   <widget class="QLabel" name="tip1_Label">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>570</y>
      <width>391</width>
      <height>111</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>黑体</family>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="wordWrap">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QGroupBox" name="step2_GroupBox">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>130</y>
      <width>400</width>
      <height>400</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">#step2_GroupBox{background: #E4F4FC;
border-radius: 12px;}</string>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QLabel" name="step2Picture_Label">
     <property name="geometry">
      <rect>
       <x>7</x>
       <y>60</y>
       <width>386</width>
       <height>280</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>690</x>
      <y>50</y>
      <width>130</width>
      <height>50</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>黑体</family>
      <pointsize>25</pointsize>
     </font>
    </property>
    <property name="text">
     <string>步骤二</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="tip2_Label">
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>580</y>
      <width>391</width>
      <height>91</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>黑体</family>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="wordWrap">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>50</y>
      <width>130</width>
      <height>50</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>黑体</family>
      <pointsize>25</pointsize>
     </font>
    </property>
    <property name="text">
     <string>步骤一</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QGroupBox" name="step3_GroupBox">
    <property name="geometry">
     <rect>
      <x>1040</x>
      <y>130</y>
      <width>400</width>
      <height>400</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">#step3_GroupBox{background: #E4F4FC;
border-radius: 12px;}</string>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QLabel" name="step3Picture_Label">
     <property name="geometry">
      <rect>
       <x>7</x>
       <y>60</y>
       <width>386</width>
       <height>280</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </widget>
   <widget class="QLabel" name="tip3_Label">
    <property name="geometry">
     <rect>
      <x>1050</x>
      <y>580</y>
      <width>391</width>
      <height>91</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>黑体</family>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="wordWrap">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>1160</x>
      <y>50</y>
      <width>130</width>
      <height>50</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>黑体</family>
      <pointsize>25</pointsize>
     </font>
    </property>
    <property name="text">
     <string>步骤三</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
